cromwell-helm/templates/cromwell-config.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "app.fullname" . }}-cromwell-config
+data:
+  {{ .Values.cromwellConfig.configFile }}: |-
+    include required(classpath("application"))
+
+    google {
+
+      application-name = "cromwell"
+
+      auths = [
+        {
+          name = "application-default"
+          scheme = "application_default"
+        }
+      ]
+    }
+
+    engine {
+      filesystems {
+        gcs {
+          // Will be used if the wdl interacts with files outside of the context of a backend.
+          auth = "application-default"
+          project = "{{ .Values.cromwellConfig.gcsProject }}"
+        }
+      }
+    }
+
+    backend {
+      default = "Local"
+      providers {
+        PAPIv2 {
+          actor-factory = "cromwell.backend.google.pipelines.v2beta.PipelinesApiLifecycleActorFactory"
+          config {
+            // Google project
+            project = "{{ .Values.cromwellConfig.gcsProject }}"
+
+            // Base bucket for workflow executions
```

Comment:
Is this the user's bucket, or something that Leo provides?

Reply:
It's the workspace bucket, which Leo will be able to know - and therefore tell us - about

Reply:
I have forgotten… what is stored in this bucket? It surely isn't the outputs of the workflows.

Reply:
@cjllanwarne ^

Reply:
Discussed in person - yes this is the location where the app user will store their workflow outputs.